playbook/app/pb_kits/playbook/pb_date_picker/_date_picker.tsx

```diff
@@ -67,7 +67,7 @@ const DatePicker = (props: DatePickerProps) => {
     hideIcon = false,
     hideLabel = false,
     id,
-    inLine = true,
+    inLine = false,
```

Comment:
This "could" potentially be a breaking change. Adding the "breaking change" label, and holding off until more testing is done.